src/lint/collect-spelling-diagnostics.ts

```diff
@@ -56,6 +56,10 @@ let matchers = [
     pattern: /(?<=\b[Ss]teps? )\d/gu,
     message: 'prefer using labeled steps and <emu-xref> tags over hardcoding step numbers',
   },
+  {
+    pattern: /(?<=\S)  +(?! |<\/t(d|h)>)/gu,
+    message: 'multiple consecutive spaces are not allowed',
```

Comment:
```suggestion
    message: 'multiple consecutive non-indentation spaces are not allowed',
```

Reply:
That's more precise, but I don't think the extra verbiage is necessary, since this will only warn on non-indentation spaces.